src/labels_by_user_input.py

```diff
@@ -1,21 +1,52 @@
+import re
+
 from src.utils import get_labels
 
 
-LABEL_VERIFIED = "Verified"
+LABELS_SET = {
+    "hold": "do-not-merge/on-hold",
+    "verified": "Verified",
+    "bug": "bug",
+}
+
+UNSUPPORTED_LABELS = f"""
+You're trying to add/remove an unsupported label.
+Supported labels - {[f"/{key} >> {LABELS_SET[key]}" for key in LABELS_SET]}
+To remove a label - /unLABEL_NAME
+"""
+
+
+def remove_label(pull, label):
+    print(f"Remove {label} from {pull.title}")
+    pull.remove_from_labels(label)
 
 
 def remove_verified_label(pull):
     labels = get_labels(pull=pull)
-    if LABEL_VERIFIED in labels:
-        print(f"Remove {LABEL_VERIFIED} from {pull.title}")
-        pull.remove_from_labels(LABEL_VERIFIED)
+    verified = LABELS_SET["verified"]
+    if verified in labels:
+        remove_label(pull=pull, label=verified)
 
 
 def labels_by_user_input(data, pull):
     body = data["comment"]["body"]
-    if "/verified" in body and LABEL_VERIFIED not in get_labels(pull=pull):
-        print(f"Adding {LABEL_VERIFIED} to {pull.title}")
-        pull.add_to_labels(LABEL_VERIFIED)
+    user = data["comment"]["user"]["login"]
+    issue_labels = get_labels(pull=pull)
+    comment_labels = re.findall("/\\S*", body)
+
+    for label in comment_labels:
+        label = label.lstrip("/").lower()
+        stripped_label = label.lstrip("un")
+
+        if stripped_label in LABELS_SET:
+
+            if label.startswith("un") and stripped_label in issue_labels:
+                remove_label(pull=pull, label=stripped_label)
+
+            elif label not in issue_labels:
+                print(f"Adding {label} to {pull.title}")
+                pull.add_to_labels(label)
 
-    if "/unverified" in body:
-        remove_verified_label(pull=pull)
+        else:
+            print(UNSUPPORTED_LABELS)
+            pull.create_comment(body=f"Hey @{user},{UNSUPPORTED_LABELS}")
```

Comment:
I think it's better to check this first and continue if we don't need to do anything with the current label.

this way you need to check it only once, now you check it twice in line 43 and 46